examples/image_classifier/README.md

```diff
@@ -15,14 +15,23 @@
 3. Python is installed on your local machine
 
 #### Instructions to run
-1. Install the requirements using command:
+1. Create a python virtual environment to run the example and activate it
+    ```sh
+    python -m venv env_example_imageclassifier
```

Comment:
Might be wise to add this directory to the .gitignore as well

Reply:
Done.